cd2024bfs2g1-frontend/src/main/ngx/src/assets/i18n/en.json

```
@@ -224,10 +224,10 @@
   "DATE_RANGE": "Select dates",
   "SEPARATOR": " to ",
   "COMBO_DIFFICULTY": "Select difficulty",
-  "EASY": "EASY",
-  "INTERMEDIATE": "INTERMEDIATE",
-  "HARD": "HARD",
-  "EXTREME": "EXTREME",
+  "EASY": "Easy",
+  "INTERMEDIATE": "Intermediate",
+  "HARD": "Hard",
+  "EXTREME": "Ectreme",
```

```suggestion
  "EXTREME": "Extreme",
```

non sei se estas traduccións tamen afectan as graficas

---

Tienen todas las traducciones en minúscula, menos los nombres de las provincias, así que es más coherente así